src/pages/gen2/build-a-backend/data/connect-to-existing-database/index.mdx

```diff
@@ -0,0 +1,410 @@
+export const meta = {
+  title: 'Connect your app to existing MySQL and PostgreSQL database.',
+  description:
+    'Learn how to connect your app to existing MySQL and PostgreSQL database.'
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      meta
+    }
+  };
+
+}
+
+Amplify's native integration supports any MySQL or Postgres database, no matter if they're hosted on AWS within a VPC or outside of AWS with a 3rd party hosted database provider.
+
+You must have these information handy for your database:
+
+- Database hostname
+- Database port
+- Database username
+- Database user password
+- Database name
+
+## Step 1 - Set secrets for database connection
+
+First, provide all the database connection information as secrets, you can use the Amplify sandbox's secret functionality to set them or go to the Amplify console to set secrets in a shared environment:
+
+<BlockSwitcher>
+<Block name="Sandbox">
+
+```bash
+npx amplify sandbox secret set MYSQL_CONNECTION_STRING
+# postgres://rene:password123@hohuoeatsuhsaoetu.eoauoaeum/main
+```
+
+</Block>
+
+<Block name="Amplify console"></Block>
+
+</BlockSwitcher>
+
+## Step 2 - Generate database schema
+
+Run the following command to generate the Data schema with your database connection information.
+
+```bash
+npx amplify generate schema-from-database --connection-uri-secret MYSQL_CONNECTION_STRING --out amplify/data/schema.rds.ts
+```
+
+<Callout info title="VPC connection">
+
+If your database is behind a VPC, we'll autodetect the VPC configuration based on your connection string. This happens via a hostname lookup within your account. The sandbox profile in your account needs to have XYZ permissions to do a VPC lookup and connect to the VPC. (If your database is in a private subnet, we'll provision a Lambda to retrieve the database schema in order for us to generate the TypeScript data model.) TODO: Chris / Ed to connect on how best to implement this.
+
+</Callout>
+
+This creates a new **schema.rds.ts** with a schema reflecting the types of your database. Import the schema to your **amplify/data/resource.ts** file and
+
+```ts
+import { a, defineData } from '@aws-amplify/backend';
+// highlight-next-line
+import { schema as auroraSchema } from './schema.rds.ts';
```

Comment:
Is there any particular reason for using `auroraSchema` versus a more general `sqlSchema`?